docs/ops.html

```diff
@@ -686,7 +686,7 @@
 
   <h3><a id="monitoring" href="#monitoring">6.6 Monitoring</a></h3>
 
-  Kafka uses Yammer Metrics for metrics reporting in both the server and the client. This can be configured to report stats using pluggable stats reporters to hook up to your monitoring system.
+  Kafka uses Yammer Metrics for metrics reporting in the server and a custom metrics registry in the client. Both expose metrics via JMX and can be configured to report stats using pluggable stats reporters to hook up to your monitoring system.
```

Comment:
Yes, I believe that was the main motivation. One thing that makes things a little more complicated is that Kafka Metrics is only used by the Java clients. I'd also replace "Custom Metrics registry" with "Kafka Metrics".